tests/upgrade_script/static/basic.log

```diff
@@ -0,0 +1,8 @@
+
+[INFO]    falco will be enabled. Change "falco.enabled" to "false" if you want to disable it (default for 1.0)
+
+[INFO]    Replacing tail-db/tail-containers-state.db to tail-db/tail-containers-state-sumo.db
```

Comment:
Hm I'm not sure we should output anything about fluentbit if there were no changes for it 🤔